<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>prime</class>
 <widget class="QWidget" name="prime">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
background-color: rgb(18, 78, 102);


</string>
  </property>
  <widget class="QPushButton" name="clean_but">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>390</y>
     <width>391</width>
     <height>291</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    font: 40pt &quot;Ubuntu&quot;;
    background-color: transparent;
    border-radius: 20px;
	image: url(:/images/unselectedbutton.png);
    border: none;
    font-weight: bold;
color:white;
}

QPushButton:focus {
    outline: none;
    border: none;
}
</string>
   </property>
   <property name="text">
    <string>CLEAN</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Tune_but">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>391</width>
     <height>291</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    font: 40pt &quot;Ubuntu&quot;;
    background-color: transparent;
    border-radius: 20px;
	image: url(:/images/unselectedbutton.png);
    border: none;
    font-weight: bold;
color:white;
}

QPushButton:focus {
    outline: none;
    border: none;
}
</string>
   </property>
   <property name="text">
    <string>TUNE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="prime1_but">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>391</width>
     <height>291</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    font: 40pt &quot;Ubuntu&quot;;
    background-color: transparent;
    border-radius: 20px;
	image: url(:/images/unselectedbutton.png);
    border: none;
    font-weight: bold;
color:white;
}

QPushButton:focus {
    outline: none;
    border: none;
}
</string>
   </property>
   <property name="text">
    <string>PRIME</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>10</y>
     <width>861</width>
     <height>711</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_6">
    <property name="styleSheet">
     <string notr="true">
background-color: rgb(18, 78, 102);


</string>
    </property>
    <attribute name="title">
     <string>Tab 1</string>
    </attribute>
    <widget class="QLabel" name="process_lab_2">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>20</y>
       <width>351</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font:25pt &quot;Ubuntu&quot;;
background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
border-radius:20px;
border:none;font:bold;</string>
     </property>
     <property name="text">
      <string>     PROCESS </string>
     </property>
    </widget>
    <widget class="QPushButton" name="start_prime_but_2">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>550</y>
       <width>321</width>
       <height>121</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
font:20pt &quot;Ubuntu&quot;;
image: url(:/images/g4210.png);
background-color:transparent;
color: rgb(255, 255, 255);
border-radius:20px;
border:none;font:bold;
}
QPushButton:focus {
    outline: none;
    border: none;
}
</string>
     </property>
     <property name="text">
      <string>START PRIME</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="start_check_2">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>160</y>
       <width>721</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QCheckBox::indicator {
    width: 30px;
    height: 30px;
}
QCheckBox {
    font-size: 30px;

}
</string>
     </property>
     <property name="text">
      <string>Start Irrigation Valve</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="wait_Check_2">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>260</y>
       <width>751</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QCheckBox::indicator {
    width: 30px;
    height: 30px;
}
QCheckBox {
    font-size: 30px;

}
</string>
     </property>
     <property name="text">
      <string>Wait For Chamber To Be Filled</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="motor_Check_2">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>70</y>
       <width>381</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QCheckBox::indicator {
    width: 30px;
    height: 30px;
}
QCheckBox {
    font-size: 30px;

}
</string>
     </property>
     <property name="text">
      <string>Start Motor</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="done_Check_2">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>370</y>
       <width>321</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QCheckBox::indicator {
    width: 30px;
    height: 30px;
}
QCheckBox {
    font-size: 30px;

}
</string>
     </property>
     <property name="text">
      <string>Done</string>
     </property>
    </widget>
    <widget class="QProgressBar" name="progressBar_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>480</y>
       <width>761</width>
       <height>51</height>
      </rect>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_5">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>530</y>
       <width>171</width>
       <height>141</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
image: url(:/images/nextimg.png);
background-color:transparent;
border:none;
}
QPushButton:focus {
    outline: none;
    border: none;
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_8">
    <property name="styleSheet">
     <string notr="true">
background-color: rgb(18, 78, 102);


</string>
    </property>
    <attribute name="title">
     <string>Page</string>
    </attribute>
    <widget class="QLabel" name="warning_lab_2">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>60</y>
       <width>431</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font:25pt &quot;Ubuntu&quot;;
background-color: rgb(36, 31, 49);
color: rgb(255, 255, 255);
border-radius:20px;
border:none;
font:bold;</string>
     </property>
     <property name="text">
      <string>   BEFORE CLEANING</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="Instruction_lab_2">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>210</y>
       <width>741</width>
       <height>281</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
font:20px;border:none;
background-color: transparent;
</string>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:20px; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;1.Disconnect the Irrigation/Aspiration connectors from handpiece&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:14pt; font-weight:600;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;2.Connect the irrigation,aspiration connectors together&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:14pt; font-weight:600;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;3.Disconnect the saline connection from irrigation.&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:14pt; font-weight:600;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;4.Disconnect the tubing from irriagtion valve.&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:14pt; font-weight:600;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;5.dip the irrigation connector in distilled water bowl&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="begin_clean_but_2">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>520</y>
       <width>331</width>
       <height>141</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
font:20pt &quot;Ubuntu&quot;;
background-color: transparent;
image: url(:/images/g4210.png);
color: rgb(255, 255, 255);
font:bold;
border-radius:20px;
border:none;
}
QPushButton:focus {
    outline: none;
    border: none;
}
</string>
     </property>
     <property name="text">
      <string>Begin Clean</string>
     </property>
    </widget>
    <widget class="QLabel" name="warning_img_2">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>60</y>
       <width>31</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/images/warning1.png);
BACKGROUND-COLOR:TRANSPARENT;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>130</y>
       <width>651</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 20pt &quot;Ubuntu&quot;;
font:bold;
background-color:transparent;</string>
     </property>
     <property name="text">
      <string> Instructions To Be Followed:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_8">
     <property name="geometry">
      <rect>
       <x>700</x>
       <y>540</y>
       <width>151</width>
       <height>141</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
image: url(:/images/backimg.png);
border:none;
background-color:transparent;
}
QPushButton:focus {
    outline: none;
    border: none;
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
